src/labels/CustomLabels.labels

```diff
@@ -2221,15 +2221,20 @@
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Spinner text when Elevate Commitment is being created</shortDescription>
-        <value>Creating Elevate recurring payment...This may take a few minutes.</value>
+        <value>Saving Elevate recurring payment...This may take a few minutes.</value>
     </labels>
     <labels>
         <fullName>RD2_EntryFormSaveCommitmentFailedMessage</fullName>
         <categories>Recurring-Donations, Entry-Form</categories>
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Message displayed as prefix to errors returned from Elevate API</shortDescription>
-        <value>However, an Elevate recurring payment could not be created due to the following errors:</value>
+        <value>A recurring payment has been created in Elevate. Elevate Recurring Id: {0}.
```

Comment:
A new line is added in the custom label and `lightning-formatted-text` used to display new line and bulletin point. The `<br/>` and other html tags were not used, but could be with  `lightning-formatted-rich-text` since the error log in the email and in NPSP Settings > Error Log would contain tags and the message would not be user friendly.